sagemaker-pipelines/tabular/abalone_build_train_deploy/sagemaker-pipelines-preprocess-train-evaluate-batch-transform.ipynb

```diff
@@ -102,7 +102,7 @@
     "\n",
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #3.    !{sys.executable} -m pip install "sagemaker==2.91.1"
</pre><p>Can all of the notebooks use <code>sagemaker&gt;=2.91.1</code> so they continue to use newer versions in the future? </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/3424/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>

Reply:
<p>Thanks for the review!</p><p>The reason to lock the notebook version is:</p><ul><li>currently some usages of Parameter are not correct in several pipeline notebooks and we will have several updates on Python SDK to prevent such behaviors, which means these updates are not backward compatible.</li><li>To avoid these incompatible changes to break the notebook build, we'll temporarily lock the sdk version.</li><li>Plus we have a plan to update all pipeline notebooks according to our new sdk features/fixes, so we can change it to be <code>sagemaker&gt;=2.91.1</code> at that time. </li></ul>